DisguiseUnityRenderStream/Runtime/NativeRenderingPlugin.cs

```diff
@@ -216,7 +216,7 @@ public EventDataPool()
             }
         }
         
-        public void Dispose()
+        protected override void Dispose()
```

Comment:
This a workaround for `NativeArray` complaining about `Dispose()` not being called when you perform a domain reload, right? It's super-annoying.

Reply:
Yes exactly